docs/Configuration/configuration-files/README.md

```diff
@@ -531,6 +531,8 @@ Use with caution as this may break Fleet ingestion of hosts data.
     detail_query_overrides:
       # null allows to disable the "users" query from running on hosts.
```

Comment:
I realize your addition uses wording consistent with what was already here, but I think this should change to:
```suggestion
      # null disables the "users" query from running on hosts.
```